### PR TITLE
VotePanelContent: add no token cost to info card

### PR DESCRIPTION
**Pull request by AquiGorka**
Reference: https://github.com/aragonone/product/issues/30

### CODE REVIEW

File: apps/voting/app/src/components/VotePanelContent.js [hunk 1]
```diff
@@ -340,6 +333,24 @@ class VotePanelContent extends React.Component {
...
+    align-items: flex-start;
+  }
+`
+
+const NoTokenCost = () => (
+  <p css="margin-top: 10px">
+    Performing this action will{' '}
+    <span css="font-weight: bold">not transfer out</span> any of your tokens.
+    You’ll only have to pay for the{' '}
+    <SafeLink href="https://ethgas.io/" target="_blcank">
```
Comment (2color) [inline apps/voting/app/src/components/VotePanelContent.js]:
```suggestion
    <SafeLink href="https://ethgas.io/" target="_blank">
```

File: apps/voting/app/src/components/VotePanelContent.js [hunk 2]
```diff
@@ -340,6 +333,24 @@ class VotePanelContent extends React.Component {
...
+      ETH fee
```
Comment (2color) [inline apps/voting/app/src/components/VotePanelContent.js]:
```suggestion
      ETH gas fee
```

Comment (AquiGorka) [reply apps/voting/app/src/components/VotePanelContent.js]: cc @luisivan and @dizzypaty for copy suggestion here.

File: apps/voting/app/src/components/VotePanelContent.js [hunk 3]
```diff
@@ -313,23 +303,26 @@ class VotePanelContent extends React.Component {
...
-                          , since it was your balance at the beginning of the
-                          vote {this.renderBlockLink(snapshotBlock)}.
-                        </p>
+                        <div>
+                          <p>
+                            You will cast your vote with{' '}
+                            {userBalance === null
+                              ? '… tokens'
+                              : pluralize(userBalance, '$ token', '$ tokens')}
+                            , since it was your balance at the beginning of the
```
Comment (sohkai) [inline apps/voting/app/src/components/VotePanelContent.js]: "it was your balance at the beginning of the vote created on 03/27/19 at 12:01 UTC"

Is quite the mouthful; perhaps we can revise this copy a bit @dizzypaty?

Suggestion:

"it was your balance when the vote was created (03/27/19 at 12:01 UTC)"

**Pull request merged**